server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/UserTester.java

```diff
@@ -105,14 +105,14 @@ public final User generateAdministratorOnDefaultOrganization(Consumer<CreateRequ
   @SafeVarargs
   public final User generateMember(Organizations.Organization organization, Consumer<CreateRequest.Builder>... populators) {
     User user = generate(populators);
-    session.wsClient().organizations().addMember(organization.getKey(), user.getLogin());
+    session.wsClient().organizationsOld().addMember(organization.getKey(), user.getLogin());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "organizationsOld"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)